Documentation/ApiOverview/RequestLifeCycle/RequestAttributes/CurrentContentObject.rst

```diff
@@ -30,3 +30,4 @@ Example:
      * @var \TYPO3\CMS\Frontend\ContentObject\ContentObjectRenderer $currentContentObject
      */
     $currentContentObject = $request->getAttribute('currentContentObject');
+    $curTtContentUid = $currentContentObject->data['uid'];
```

Comment:
Actually, I would avoid naming it `$curTtContentUid` - that is bad to read, instead `$currentContentUid` - there is no need to add `tt`.